translation-web/src/main/java/eu/europeana/api/translation/config/TranslationApiAutoconfig.java

```diff
@@ -135,6 +137,18 @@ public TranslationServiceProvider getTranslationServiceProvider() {
     this.translationServiceConfigProvider = new TranslationServiceProvider();
     return this.translationServiceConfigProvider;
   }
+  
+  /*
+   * For windows use this link for help: https://www.baeldung.com/spring-data-redis-properties
+   * I.e. install the docker image for Redis (it can be without the password), and put the properties (e.g. spring.redis.host) 
+   * in the translation.(user.)properties file
+   */
+  @Bean
```

Comment:
I think we would still need the beans for the LettuceConnectionFactory and RedisTemplate, because e.g. for the proper initialization of the lettuce factory, spring internally calls the methods such as afterProperties(), which is called after the bean creation (@Bean annotation needed). Otherwise we would need to call those methods manually which is not the best option. Please see also my comment in the code at the bean method.